pkg/archive/archive.go

```diff
@@ -633,6 +647,11 @@ func TarWithOptions(srcPath string, options *TarOptions) (io.ReadCloser, error)
 				}
 
 				if err := ta.addTarFile(filePath, relFilePath); err != nil {
+					if err == errSkipFilePermissions {
+						// GH18120. Note next line is deliberately NOT logrus or %q
+						fmt.Printf("Skipping '%s' - Permission was denied\n", filePath)
```

Comment:
In other words, same as https://github.com/docker/docker/blob/master/builder/dockerfile/evaluator.go#L109 for the "Step" line.